Lib/_opcode_metadata.py

```diff
@@ -31,75 +31,3 @@
         "STORE_SUBSCR_DICT",
         "STORE_SUBSCR_LIST_INT",
     ],
-    "SEND": [
```

Comment:
That this file changed surprises me. Maybe this points to a problem?

Reply:
It didn't change originally, I think only after https://github.com/python/cpython/pull/106716/commits/178e3009234e49102d2d8058504f9c25eb374a06 ?